services/backend/src/service/protected_area.js

```diff
@@ -31,6 +31,19 @@ module.exports = (paPersistence, seService) => {
       paPersistence.findCategoriesByBinaryProtected(binaryProtected)
     ),
 
+    /**
+     * Get the binary protected value for the given category name
+     *
+     * @param {String} categoryName protected area category name
+     *
+     * @returns {Object} binary protected value
+     *
+     */
+    getBinaryProtectedByCategory: async (categoryName) => {
+      const data = await paPersistence.findBinaryProtectedByCategory(categoryName);
+      return data ? data[0] : null;
```

Comment:
I think that if data or data[0] is null it should return an error, because that category doesn't exists. Besides, returning `null` would mean there is no error and the process wherever this is called from should continue

Reply:
Done